packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -296,14 +296,30 @@ const Breakouts = WebexPlugin.extend({
     });
   },
 
+  /**
+   * get current breakout is in progress or not
+   * @returns {boolean}
+   */
+  getIsBreakoutInProgress() {
```

Comment:
i would just change the name to isBreakoutInProgress 

we normally only add get when there is a API needed

Reply:
@arun3528 thanks, I will change it.